File: beagle-ui/src/main/java/br/com/zup/beagleui/framework/widget/layout/Horizontal.kt
package br.com.zup.beagleui.framework.widget.layout

import br.com.zup.beagleui.framework.widget.core.Flex
import br.com.zup.beagleui.framework.widget.core.Widget

data class Horizontal(
    val children: List<Widget>,
    val flex: Flex = Flex(),
    val reversed: Boolean = false
) : Widget

File: Sources/BeagleUI/Sources/Layout/Horizontal.swift
//
//  Horizontal.swift
//  BeagleUI
//
//  Created by Daniel Tes on 12/09/19.
//  Copyright © 2019 Daniel Tes. All rights reserved.
//

public struct Horizontal: Widget {
    
    let children: [Widget]
    let flex: Flex?
    let reversed: Bool
    
    init(
        children: [Widget] = [],
        flex: Flex? = nil,
        reversed: Bool = false
    ) {
        self.children = children
        self.flex = flex
        self.reversed = reversed
    }
    
}
